typed/validate.go

```diff
@@ -205,15 +219,15 @@ func listItemToPathElement(list schema.List, index int, child value.Value) (fiel
 	return pe, nil
 }
 
-func (v validation) doList(t schema.List, value value.Value) (errs []ValidationError) {
+func (v validation) doList(t schema.List, value value.Value) (errs ValidationErrori) {
```

Comment:
`ValidationErrori`?

Reply:
Huh, I thought I pushed the fix for this before making the PR, weird.